ci/kubetest/test_kafka_external.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+from helpers.utils import (
+    cleanup_k8s,
+    create_namespace_if_not_exists,
+    install_chart,
+    install_external_kafka,
+    is_posthog_healthy,
+    wait_for_pods_to_be_ready,
+)
+
+VALUES_YAML = """
+cloud: local
+
+kafka:
+  enabled: false
+
+externalKafka:
+  brokers: "kafka0:9092"
```

Comment:
Non-blocking

Maybe not relevant, but can we add a test that uses comma separated list here.

(I see we're doing some manipulation for the dependency check but that's only for internal, which I assume is always going to be a single clusterip? So that isn't an issue, but for e.g. MSK we'll have a list.)